project.code-workspace

```diff
@@ -77,7 +72,7 @@
     "terminal.integrated.fontFamily": "Hack Nerd Font",
     "window.title": "${activeEditorMedium}${separator}${rootName}${separator}${rootPath}",
     "window.zoomLevel": 0,
-    "workbench.colorTheme": "Noctis Azureus",
+    "workbench.colorTheme": "Better Solarized Dark Italics",
```

Comment:
Is this mandatory or suggested. I use a light IDE theme for example

Reply:
I will explain later but these are the defaults across the project and should not be changed. However, you are free to make any local changes you want but these shouldn't be pushed

Reply:
We'll try to come up with an idea that ensures 
1) Content of this file remains as is
2) It can be easily customisable by us on our laptops

Reply:
@terado & @dehe1 I've created this issue so it is not forgotten https://github.com/nhsd-exeter/make-devops/issues/58